_rules/device-motion-disabled-c249d5.md

```diff
@@ -30,19 +30,20 @@ The rule applies to an [HTML document][] with an associated [Window object][] th
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] to prevent the outcome of the event is available in the same [web page][], or in another [web page][] that is [linked][hyperlink] from the [web page][] of the test target.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] to prevent the results of the event.
```

Comment:
I'm not completely sure about "prevent". See also https://github.com/act-rules/act-rules.github.io/pull/873#discussion_r423593131

Replacing the effect of device motion by an other effect would "prevent the results" of the event but still be pretty bad (as there is now a new device motion event). So for example, a slider where any motion increase it, and a button toggles to any motion decrease the slider, the button would "prevent the results" of the event and thus pass the rule.

I think what we went is that after activating the instrument, "nothing" happens when the event is fired. But that need a bit of definition of "nothing" (change in the DOM or visible content, playing media, redirecting, …) Note that the definition doesn't need to be too strict: if the rule pass when "disabling" still allow sound to be played, we're in one of the "further testing needed" case. Bad (especially if too broad), but not too bad.

Reply:
@Jym77 I don't think the problem here is prevent (definition of prevent is ""to keep from happening or existing"). The problem is what we are preventing. I agree with you when you say that if we replace the result of one event with another result, then we are preventing the original result from happening. I guess what we want in this instance is to prevent any results of the event from happening (i.e. disable the event). I'll rephrase it accordingly.